main.go

```diff
@@ -144,6 +175,29 @@ func main() {
 	} else {
 		subscriberQoS = lvl
 	}
+	var ca, cert, key []byte
+	if err := validateTLSFiles(argCafile, argKey, argCert); err != nil {
+		fmt.Fprintln(os.Stderr, err)
+		os.Exit(1)
+	} else {
+		var err error
+		ca, err = ioutil.ReadFile(*argCafile)
+		if err != nil {
+			fmt.Fprintln(os.Stderr, err)
+			os.Exit(1)
+		}
+		cert, err = ioutil.ReadFile(*argCert)
+		if err != nil {
+			fmt.Fprintln(os.Stderr, err)
+			os.Exit(1)
+		}
+		key, err = ioutil.ReadFile(*argKey)
+		if err != nil {
+			fmt.Fprintln(os.Stderr, err)
+			os.Exit(1)
+		}
+		*argSkipTLSVerification = false
```

Comment:
This change does not have the intended effect. Your TLs config is added to the worker options after this flag is copied to the publisher options. Please just set InsecureSkipVerify to false as written in my previous comment.

Reply:
my bad, I misunderstood the request.